airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -1,2 +1,27 @@
+function getSlug() {
```

Comment:
This whole file needs to be wrapped in something like `$(function() { ... })` so no javascript magic happens until everything has been loaded. 

And ideally I would use a closure so you don't add `getSlug()` to the `window` global scope. 

You can do this pattern if you like:
```
var PopcornIntegration = (function() {
    function getSlug() {
    }
    return {
        init: function() {
          PopcornEditor.listen(...)
        }
    }
});

$(function() {
    PopcornIntegration.init();
});
```